src/components/Info/Info.tsx

```diff
@@ -1,17 +1,19 @@
 import * as React from 'react'
+import classNames from 'classnames'
 import { Popup } from 'decentraland-ui'
+import { InfoIcon } from 'components/InfoIcon'
 import { Props } from './Info.types'
 import './Info.css'
 
 export default class Info extends React.PureComponent<Props> {
   render() {
-    const { className = '', content = '' } = this.props
+    const { className, content = '' } = this.props
     return (
       <Popup
         className="info-popup"
         content={content}
         position="top center"
-        trigger={<i className={`Info ${className}`} />}
+        trigger={<InfoIcon className={classNames(className, 'infoIcon')} />}
```

Comment:
are we using the infoIcon class? and, also, why is camelCased?

Reply:
We're! it's giving this `InfoIcon` the `cursor: pointer;` property. I'll change it so it's not camel cased.